src/views/Opportunities/OpportunityComponents/OpportunityForm.jsx

```diff
@@ -91,6 +91,18 @@ const OpportunityForm = (props) => {
         setEmployerOptions(data);
       });
     }
+
+    if (props.id) {
+      const searchedEmployer = employerOptions.find((option) => option.label === props.employer.name);
+      if (!searchedEmployer) {
+        const newvalue = {
+          key: props.employer.name,
+          label: props.employer.name,
+          value: Number(props.employer.id),
+        };
+        employerOptions.push(newvalue);
+      }
+    }
```

Comment:
can't we put this after `setEmployerOptions(data)` line? I don't htink we need to check for `props.id` again as we're already checking for `props.institution` above.

Also, the `filterEmployer` looks like an async function so it's likely that the code you've written may execute before the browsers gets the API response from filterEmployer.

Reply:
when i am doing console.log( props.institution) it is showing ```undefined```

Reply:
![image](https://user-images.githubusercontent.com/117888235/233687858-9976c5af-676f-4912-80eb-7db6965235f1.png)
it is showing undefined
![image](https://user-images.githubusercontent.com/117888235/233688033-85c692ec-abd4-4389-acf2-f290b8124386.png)

Reply:
so i put code here below the setEmployerOptions
![image](https://user-images.githubusercontent.com/117888235/233690942-ff7c55e4-fa47-42c0-8f4f-f3eb560f7a7c.png)